intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -73,17 +73,17 @@ python scripts/run_llm.py model-path --weight_dtype int4 -p "She opened the door
 LLM one-click running script args explanations:
 | arg               | explanation                                                             |
 | --------------    | ----------------------------------------------------------------------- |
-| model           | directory containing model file or model id                 |
-| --weight_dtype  | data type of quantized weight (default: int4)         |
-| --alg           | quantization algorithm to use: sym/asym (default: sym)      |
-| --block_size    | block size (default: 32)                                    |
-| --scale_dtype   | fp32/bf16 type for scales (default: fp32)                   |
-| --compute_type  | Gemm computation data type: int8/fp32/ggml (default: ggml)  |
+| model           | directory containing model file or model id                               |
+| --weight_dtype  | data type of quantized weight (default: int4)                             |
+| --alg           | quantization algorithm to use: sym/asym (default: sym)                    |
+| --group_size    | group size (default: 32)                                                  |
+| --scale_dtype   | fp32/bf16 type for scales (default: fp32)                                 |
+| --compute_type  | Gemm computation data type: int8/fp32/bf16/ggml (default: ggml)           |
```

Comment:
what's the data type in default ggml?

Reply:
The ggml data type is the same as llama.cpp, it does not contain anything related to jblas

Reply:
fp32/bf16 type for scales (default: fp32)  => data type for scales: bf16/fp32 (default: fp32) 
compute_type => compute_dtype

Reply:
can we enable a separate parameter --use_ggml to decide? instead of mixing into compute_dtype?

Reply:
update to use_ggml